### PR TITLE
Allow to test gitlab,gitea and bitbucket committime exporters.

### LINKED ISSUE #582: Allow to test gitlab,gitea and bitbucket committime exporters
Test gitlab,gitea and bitbucket committime exporters in CI.

### PR DESCRIPTION
**Pull request by mpryc**
Enabling the gitlab,gitea and bitbucket committime exporters.

The way it works it creates binary build and annotate it with the corresponding git URL and hash. Based on that specific commit time exporter back-end is collecting timestamp of the commit from the provider PI.

Fixes #582

### CODE REVIEW

File: scripts/run-pelorus-e2e-tests [hunk 1]
```diff
@@ -169,41 +195,177 @@ sed -i.bak "s/your_org/$demo_org/g" "${DWN_DIR}/mongo-persistent.yaml"
...
+        sed -i.bak "s/#gitlab-committime@//g" "${DWN_DIR}/ci_values.yaml"
+        echo "The pelorus gitlab committime exporter has been enabled"
+    fi
+
+    # namespace must much one from ci_values.yaml
+    build_ns="gitlab-binary"
+    BINARY_BUILDS_NAMESPACES+=("${build_ns}")
+    build_uri="https://gitlab.com/mpryc/pelorus-gitlab/"
+    build_hash="b807fd8e1b2bd1755eca14960e5352fe89ff466e"
+    eval "${BINARY_BUILD_SCRIPT}" -n "${build_ns}" -b "${build_ns}" -u "${build_uri}" -s "${build_hash}"
```
Comment (KevinMGranger) [inline scripts/run-pelorus-e2e-tests:322]: eval isn't necessary here, leading with `"${BINARY_BUILD_SCRIPT}"` should work just fine. Applies to its uses elsewhere also.

File: scripts/run-pelorus-e2e-tests [hunk 2]
```diff
@@ -32,7 +37,7 @@ function download_file_from_url() {
     local file_name=$2
     local dest_folder="${3:-$SCRIPT_DIR}" # Use ./ as default dest_folder
 
-    pushd "${dest_folder}" || exit
+    pushd "${dest_folder}" || exit/var/run/konveyor/pelorus/
```
Comment (KevinMGranger) [inline scripts/run-pelorus-e2e-tests:40]: typo

Comment (mpryc) [reply scripts/run-pelorus-e2e-tests]: thanks, accidental paste

**Comment by mpryc** [comment]
@weshayutin this is ready to be reviewed. Gitea is implemented, but disabled until #592 is fixed.

**Comment by weshayutin** [comment]
looks good.. I test the binary build.. worked and the rest looks good and will be caught in CI fairly obviously.
kind: Build
apiVersion: build.openshift.io/v1
metadata:
  annotations:
    io.openshift.build.commit.id: '1234'
    io.openshift.build.source-location: 'https://wes/foo'
    openshift.io/build-config.name: westest
    openshift.io/build.number: '1'
    openshift.io/build.pod-name: westest-1-build
  resourceVersion: '98826846'
  name: westest-1

**Pull request merged**